pkg/cmd/roachtest/roachtestutil/mixedversion/mixedversion.go

```diff
@@ -180,21 +185,21 @@ var (
 	// [1] https://www.cockroachlabs.com/docs/releases/release-support-policy#current-supported-releases
 	OldestSupportedVersion = clusterupgrade.MustParseVersion("v23.1.0")
 
-	// OldestSupportedVersionVC is similar to `OldestSupportedVersion`,
-	// but applies only to virtual cluster deployments. The reason it is
-	// different for now is that v23.1.5 is the first version to ship
-	// with the ability to change the default target cluster, a core
-	// component of tests generated by the framework.
+	// OldestSupportedVersionSP is similar to `OldestSupportedVersion`,
+	// but applies only to shared-process virtual cluster deployments.
+	// The reason it is different for now is that v23.1.5 is the first
+	// version to ship with the ability to change the default target
+	// cluster, a core component of tests generated by the framework.
 	//
 	// TODO(testeng): remove this constant when 23.1 reaches EOL.
-	OldestSupportedVersionVC = clusterupgrade.MustParseVersion("v23.1.5")
+	OldestSupportedVersionSP = clusterupgrade.MustParseVersion("v23.1.5")
```

Comment:
To avoid (name) overloading, we could also consider using `external process`. After all, our unit tests still use that terminology, e.g.,

> test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.

Reply:
Given that this variable is only used in a couple places internally and it will go away when 23.1 reaches EOL (in a month or two), I'm leaning towards keeping this as-is until then.